### ISSUE TITLE
Machikane names

### ISSUE THREAD

**Issue by noccu** [FabulousCupcake/umamusume-db-translate#90]
As _Machikane Fukukitaru_ is rendered as _Matikanefukukitaru_ to comply with the 18 character name limit on race horse names, _Machikane Tannhäuser_'s name should also be rendered as _Matikanetannhauser_, as, for example, [JBIS uses](https://www.jbis.jp/horse/0000220776/)

Moreover, the naming for _Fukukitaru_ is not consistent nor correct, with _Machikanefukukita_ used in `missions.csv` and `trainer-title-requirements.csv`

Alternatively, the length limit could be ignored for both and Fukukitaru's "full" name used...

**Comment by robflop** [comment]
To add onto that: While of course the official name (as in, for the real horse) and such is important, I think the decisive point is Cygames's website for the Series. So even if there were disparities between the officially registered name and the name within the Umamusume universe, i'd go for the latter.

In this case though (and I expect it to be like this for everyone), they coincide. See here: https://umamusume.jp/character/detail/?name=matikanetannhauser

tl;dr: Yes, I agree on using Matikanetannhauser instead - and so does Cygames.

**Comment by FabulousCupcake** [comment]
No objection, I think it should be edited too
